src/pages/gallery/gallery.ts

```diff
@@ -45,10 +45,10 @@ export class GalleryPage {
   refresher = null;
 
   constructor(
-    public navCtrl: NavController,
-    public toastCtrl: ToastController,
-    public assessmentService: AssessmentService,
-    public submissionService: SubmissionService
+    private navCtrl: NavController,
```

Comment:
Hi @imkven, when do we use public and when do we use private? May I have a standard answer for this? Some parts of my code are using private, and some parts are using public, thats why I ask la, ^_^

Reply:
Please change it to `public` when you do change next time on this file.